launcher.py

```diff
@@ -1,3 +1,5 @@
+#import sys
```

Comment:
You can setup a Python interpreter which is in a virtual envoronment and then you don't have to add explicitly any additional packages to the path like that. The master branch should be as clean as possible from any user-specific code and stuff like that should not be there (even as a comment).